lib/roby/test/minitest_helpers.rb

```diff
@@ -123,45 +108,8 @@ def register_failure(e)
                 end
             end
 
-            def capture_exceptions
-                super do
-                    begin
-                        yield
-                    rescue Exception => e
-                        unless e.respond_to?(:each_original_exception)
-                            raise
-                        end
-
-                        exceptions = e.each_original_exception
-                        register_failure(e)
-
-                        # Try to be smart and to only keep the toplevel
-                        # exceptions
-                        filter_execution_exceptions(exceptions).each do |original_e|
-                            unless original_e.backtrace
-                                original_e.set_backtrace(e.backtrace)
-                            end
-                            register_failure(original_e)
-                        end
-                    end
-                end
-            end
-
-            def filter_execution_exceptions(exceptions)
-                exceptions.flat_map { |e| Roby.flatten_exception(e).to_a }.uniq
-            end
-
-            def exception_details(e, msg)
-                [
-                    msg.to_s,
-                    "Class: <#{e.class}>",
-                    "Message: <#{e.message.inspect}>",
-                    "Pretty-print:",
-                    *Roby.format_exception(e),
-                    "---Backtrace---",
-                    Minitest.filter_backtrace(e.backtrace).join("\n").to_s,
-                    "---------------"
-                ].join "\n"
+            def sanitize_exception(e)
+                e
```

Comment:
What's the point of this method?

Reply:
It's an overload of a minitest method, I'm overriding behaviior that breaks Syskit

Reply:
Updated documentation to clarify